js/code/shared/Download.svelte

```diff
@@ -72,10 +74,6 @@
 		height: 22px;
 	}
 
-	.copied {
-		color: var(--color-green-500);
-	}
```

Comment:
This style is currently not working well anyhow, so removing it shouldn't have a critical effect.
![CleanShot 2023-11-15 at 12 05 26](https://github.com/gradio-app/gradio/assets/3135397/3c2eed3f-8800-4050-813f-7714b9354178)